Extensions/xkit_patches.js

```diff
@@ -634,20 +683,41 @@ XKit.extensions.xkit_patches = new Object({
 					});
 					return this.cssMap;
 				},
+
 				keyToClasses: function(key) {
 					if (!this.cssMap || !this.cssMap.hasOwnProperty(key)) {
 						return;
 					}
 					return this.cssMap[key];
 				},
+
 				keyToCss: function(key) {
 					const classes = this.keyToClasses(key);
 					if (!classes) {
 						return;
 					}
 					return classes.map(cls => '.' + cls).join(', ');
 				},
+
+				/**
+				 * Given any number of keys, turns them into a css selector
+				 * where key[0] contains key[1], etc.
+				 *
+				 * Because CSS commas always take precedence over spaces, and
+				 * :is has poor browser support, this requires enumerating all
+				 * possible combinations of `key[0] key[1]` and then separating
+				 * them with commas.
+				 *
+				 * @param {Array<String>} keys - the cssMap keys to combine.
```

Comment:
As far as I can tell from a quick search the more common way of doing rest arguments is `{...String}`